config/translation.php

```diff
@@ -87,7 +87,7 @@
     | All available locales that can be selected.
     |
     */
-    "available_locales" => ["fa", "en", "ar", "tr"],
+    "available_locales" => explode(',', env('SITE_AVAILABLE_LOCALES',)),
```

Comment:
You can pass 'fa,en' as the second parameter to the env function to ensure if the env var does not exist, your application does not crash.